### DISCUSSION TITLE
Welcome to gollum Discussions!

### DISCUSSION THREAD

**Discussion by dometto** [gollum/gollum#1744] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by darless** [comment]
Hello all, I'm @darless1. I am a big believer in OSS and am always excited to learn new languages and functionality. Thank you to the members for the work on this wiki. 

What I have done in this community:

- Switch to Github Actions from Travis #1729 
- Working on getting Docker to be within the repository. #1732
